### ISSUE TITLE
iobroker admin gui error

### ISSUE THREAD

**Issue by BenAhrdt** [ioBroker/ioBroker.admin#1360]
Manchmal wird beim Öffnen der Objekte dies angezeigt:

![image](https://user-images.githubusercontent.com/92173565/157386968-32c4e732-f4cd-4da4-a746-cec4d33b45f8.png)

Error in der console:
```
iobroker.admin@5.3.1
instrument.ts:125 [2022-03-09T06:41:58.944Z] Try to connect
instrument.ts:125 Translate: time
instrument.ts:125 Translate: timeOffset
instrument.ts:125 Translate: time
instrument.ts:125 Translate: timeOffset
:8081/#tab-objects:1 Access to script at 'http://datenspeicher:8081/static/js/2.56fe7b4a.chunk.js' from origin 'http://datenspeicher:8081' has been blocked by CORS policy: The request client is not a secure context and the resource is in more-private address space `private`.
2.56fe7b4a.chunk.js:1          Failed to load resource: net::ERR_FAILED
instrument.ts:125 ChunkLoadError: Loading chunk 2 failed.
(error: http://datenspeicher:8081/static/js/2.56fe7b4a.chunk.js)
    at Function.u.e (datenspeicher/:1:3974)
    at main.22a9d49c.chunk.js:2:1114019
    at V (6.501acaca.chunk.js:2:5666268)
    at Zi (6.501acaca.chunk.js:2:5656452)
    at jl (6.501acaca.chunk.js:2:5643113)
    at Sl (6.501acaca.chunk.js:2:5643041)
    at Cl (6.501acaca.chunk.js:2:5642904)
    at zl (6.501acaca.chunk.js:2:5639870)
    at 6.501acaca.chunk.js:2:5589364
    at t.unstable_runWithPriority (6.501acaca.chunk.js:2:5672574)
    at Ga (6.501acaca.chunk.js:2:5589141)
    at Ka (6.501acaca.chunk.js:2:5589309)
    at Za (6.501acaca.chunk.js:2:5589244)
    at ke (6.501acaca.chunk.js:2:5660411)
    at Yt (6.501acaca.chunk.js:2:5567862)
    at HTMLDivElement.a (6.501acaca.chunk.js:2:6337076)
(anonymous) @ instrument.ts:125
```

**Comment by Visioneer3** [comment]
Ähnliches hier:

Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
instrument.ts:125 getInstances: 328
instrument.ts:125 Translate: time
instrument.ts:125 Translate: timeOffset
instrument.ts:125 TypeError: Cannot read properties of undefined (reading 'includes')
    at BaseSettingsLog.js:77:65
    at Array.forEach (<anonymous>)
    at new s (BaseSettingsLog.js:75:41)
    at zc (react-dom.production.min.js:135:172)
    at Uo (react-dom.production.min.js:181:146)
    at Zi (react-dom.production.min.js:269:427)
    at jl (react-dom.production.min.js:250:347)
    at Sl (react-dom.production.min.js:250:278)
    at Cl (react-dom.production.min.js:250:138)
    at zl (react-dom.production.min.js:243:163)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at Ga (react-dom.production.min.js:122:325)
    at Ka (react-dom.production.min.js:123:61)
    at Za (react-dom.production.min.js:122:428)
    at ke (react-dom.production.min.js:292:101)
    at Yt (react-dom.production.min.js:73:352)
    at HTMLBodyElement.a (helpers.ts:98:17)
utils.ts:95 
 POST https://sentry.iobroker.net/api/58/envelope/?sentry_key=4364315…&sentry_version=7 net::ERR_BLOCKED_BY_CLIENT
fetch.ts:57 
 POST https://sentry.iobroker.net/api/58/envelope/?sentry_key=4364315…&sentry_version=7 net::ERR_BLOCKED_BY_CLIENT
fetch.ts:57 
 POST https://sentry.iobroker.net/api/58/envelope/?sentry_key=4364315…&sentry_version=7 net::ERR_BLOCKED_BY_CLIENT
utils.ts:95 
 POST https://sentry.iobroker.net/api/58/envelope/?sentry_key=4364315…&sentry_version=7 net::ERR_BLOCKED_BY_CLIENT

**Comment by GermanBluefox** [comment]
Fixed in 5.3.2

**Comment by Feuer-sturm** [comment]
@BenAhrdt @Visioneer3 Please verify with Admin 5.3.2
